service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -47,28 +52,45 @@ public NotificationHandler(HmctsCourtLookupConfiguration hmctsCourtLookupConfigu
     public void sendNotificationToHmctsAdmin(SubmittedCaseEvent event) {
         CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
         String localAuthorityCode = (String) caseDetails.getData().get("caseLocalAuthority");
-        Map<String, String> parameters = buildEmailData(caseDetails, localAuthorityCode);
+        Map<String, String> parameters = buildHmctsSubmissionNotification(caseDetails, localAuthorityCode);
         String reference = Long.toString(caseDetails.getId());
-
         String email = hmctsCourtLookupConfiguration.getCourt(localAuthorityCode).getEmail();
+
         logger.debug(
             "Sending submission notification (with template id: {}) to {}", HMCTS_COURT_SUBMISSION_TEMPLATE, email);
 
         try {
             notificationClient.sendEmail(HMCTS_COURT_SUBMISSION_TEMPLATE, email, parameters, reference);
-        } catch (NotificationClientException ex) {
+        } catch (NotificationClientException e) {
             logger.error("Failed to send submission notification (with template id: {}) to {}",
-                HMCTS_COURT_SUBMISSION_TEMPLATE, email, ex);
+                HMCTS_COURT_SUBMISSION_TEMPLATE, email, e);
         }
     }
 
-    private Map<String, String> buildEmailData(CaseDetails caseDetails, String localAuthorityCode) {
-        Map orders =
-            Optional.ofNullable((Map) caseDetails.getData().get("orders")).orElse(ImmutableMap.builder().build());
+    @EventListener
+    public void sendNotificationToCafcass(SubmittedCaseEvent event) {
+        CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
+        String localAuthorityCode = (String) caseDetails.getData().get("caseLocalAuthority");
+        Map<String, String> parameters = buildCafcassSubmissionNotification(caseDetails, localAuthorityCode);
+        String reference = (String.valueOf(caseDetails.getId()));
+        String email = cafcassEmailLookupConfiguration.getCafcass(localAuthorityCode).getEmail();
 
-        List orderType = (List) Optional.ofNullable(orders.get("orderType")).orElse(ImmutableList.builder().build());
+        logger.debug(
```

Comment:
Can we extract couple of lines (starting with debug, ending with exception handling) into separate method and reuse across both notification method. Logging logic is same.